timelock-corruption-detection/src/main/conjure/timelock-history.yml

```diff
@@ -0,0 +1,52 @@
+types:
+  imports:
+    PaxosValue:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosValue
```

Comment:
I'm afraid this won't actually work in its current form, because `PaxosValue` is not JSON serializable.

Reply:
@Jkong verified that PaxosValue is JSON serializable.